src/gpl/src/gpuSolver.h

```diff
@@ -0,0 +1,63 @@
+#ifndef __GPUSOLVER__
+#define __GPUSOLVER__
+
+#include <cuda_runtime.h>
+#include <thrust/copy.h>
+#include <thrust/device_vector.h>
+#include <thrust/fill.h>
+#include <thrust/host_vector.h>
+#include <thrust/sequence.h>
+
+#include <Eigen/SparseCore>
+#include <memory>
+#include <vector>
+
+#include "cusolverSp.h"
+#include "cusparse.h"
+#include "device_launch_parameters.h"
+#include "odb/db.h"
+#include "utl/Logger.h"
+
+namespace utl {
+class Logger;
+}
+
+namespace gpl {
+
+using namespace std;
```

Comment:
Don't put this in a header which affects all users.  I don't see any std usage here at all.